brigade-worker/src/job.ts

```diff
@@ -163,6 +163,16 @@ export abstract class Job {
    */
   public privileged: boolean = false;
 
+  /** The account identity to be used when running this job.
+   * This is an optional way to override the build-wide service account. If it is
+   * not specified, the main worker service account will be used.
+   *
+   * Implementors may choose to ignore this value for security reasons.
```

Comment:
This wasn't clear for me at glance. Probably behind the word "Implementators", there is an implication that there could be multiple implementations of brigade-worker which runs the brigade.js?
If thats the case, perhaps "Different brigade-worker implementations may choose to ..." sounds like a better note for me.

Reply:
Good point. I will change that.